src/styles/What.css

```diff
@@ -446,4 +427,12 @@
           border-radius: 5px;
           display: inline-block;
       }
+    #wwdLeftBlurb {
+      width: 80px;
+      height: 80px;
+    }
+    #wwdRightBlurb {
+      width: 80px;
+      height: 80px;
+    }
```

Comment:
What.css: Code is clean, good incorporation of class properties for What.css, and well-organized for future additions/amendments.